api/compose/v1beta2/stack.go

```diff
@@ -230,6 +230,14 @@ type ServiceVolumeConfig struct {
 	ReadOnly bool   `json:"read_only,omitempty"`
 }
 
+func (s *StackSpec) clone() *StackSpec {
```

Comment:
Remove this function, and rely on deep copy

Reply:
Fixed